internal/cmd/manager/controller/controller.go

```diff
@@ -41,7 +42,6 @@ import (
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/multicache"
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/utils"
 	"github.com/cloudnative-pg/cloudnative-pg/pkg/versions"
-	// +kubebuilder:scaffold:imports
```

Comment:
This requires the new version of golangci right? So everyone should be running the new version ?

Reply:
No, it doesn't. It's also suitable for the old version.

Reply:
Nice!! Then let's merge it :)